provider_shopper/lib/screens/catalog.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:provider_shopper/models/cart.dart';
+
+class MyCatalog extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: CustomScrollView(
+        slivers: [
+          _MyAppBar(),
+          SliverToBoxAdapter(child: SizedBox(height: 12)),
+          SliverList(
+              delegate:
+                  SliverChildBuilderDelegate((_, index) => _MyListItem(index))),
```

Comment:
Given that this is sample code, readers might benefit more from seeing the name of the first parameter here.